README.md

```diff
@@ -0,0 +1,15 @@
+## Ravelin Data Science Test
+
+Below are some json profiles representing fictional customers from an ecommerce company. The profiles contain information about the customer, their orders, their transactions, what payment methods they used and whether the customer is fraudulent or not. Your task is one of the following.
+
+### Option A
+ * Transform the json profiles into feature vectors 
+ * Construct a model to predict if a customer is fraudulent based on their profile.
+ * Report on the models success and show what features are most important
+
+### Option B
+ * Provide exploratory analysis of the dataset.
+ * Summarise and explain the key trends in the data, providing visualisations and tabular representations as necessary.
+ * Explain what factors you think are significant and insignificant in contributing to fraud
+
+ Please use Python for this exercise. You can use whatever external software libraries you think are appropriate. You should spend no more than a couple of hours on this test. 
```

Comment:
I don't think we should say 2 hours, theres loaaaads of variables to explore here much more than the last dataset. what about no more than half a day?

Reply:
Yeah it's a bit disingenuous to say 2 hours